ui/options_advanced.ui

```diff
@@ -20,13 +20,65 @@
       <property name="spacing">
        <number>2</number>
       </property>
-      <item row="5" column="0">
-       <widget class="QCheckBox" name="recursively_add_files">
+      <item row="1" column="0">
+       <widget class="QLabel" name="label_ignore_regex">
+        <property name="sizePolicy">
+         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+          <horstretch>0</horstretch>
+          <verstretch>0</verstretch>
+         </sizepolicy>
+        </property>
         <property name="text">
-         <string>Include sub-folders when adding files from folder</string>
+         <string>Ignore file paths matching the following regular expression:</string>
+        </property>
+        <property name="wordWrap">
+         <bool>true</bool>
+        </property>
+       </widget>
+      </item>
+      <item row="8" column="0">
+       <layout class="QHBoxLayout" name="horizontalLayout_2">
+        <item>
+         <widget class="QLabel" name="label_query_limit">
+          <property name="sizePolicy">
+           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
+            <horstretch>0</horstretch>
+            <verstretch>0</verstretch>
+           </sizepolicy>
+          </property>
+          <property name="text">
+           <string>Max. number of entities to return per MusicBrainz query</string>
```

Comment:
Minor point, but should this be "Maximum number of entries..." rather than using the abbreviation?  This should not impact the width of the dialog because it will still be shorter than the previous label (I think)?